lemminx-maven/src/main/java/org/eclipse/lemminx/maven/PluginValidator.java

```diff
@@ -82,80 +79,73 @@ public Diagnostic validateConfiguration(DiagnosticRequest diagnosticRequest) {
 			// A diagnostic was already added in validatePluginResolution()
 		}
 		if (parameters.isEmpty()) {
-			return null;
+			return Optional.ofNullable(null);
```

Comment:
Wouldn't `Optional.empty()` be better here?

Reply:
Yes my bad :P